recipe/meta.yaml

```diff
@@ -51,8 +51,20 @@ test:
     - sqlalchemy.util
   requires:
     - pip
+    - pytest >=7,<8.2
+    - mypy >= 1.7.0
```

Comment:
I guess mypy isn't needed.

Reply:
Only if we skip the mypy tests. There's an option for that should that be something we desire.